common/app/navigation/NavLinks.scala

```diff
@@ -102,6 +104,8 @@ private object NavLinks {
       NavLink("Results", "/football/results", Some("football/results")),
       NavLink("Competitions", "/football/competitions", Some("football/competitions")),
       NavLink("Clubs", "/football/teams", Some("football/teams")),
+      NavLink("Euro 2020", "/football/teams", Some("football/teams")),
+      Euro2020,
```

Comment:
isn't this adding it twice?

Reply:
I think they wanted it on the left as well (so first one in this list I guess)

Reply:
Oh sorry. I was refactoring and missed that